app/component/RoutePageControlPanel.js

```diff
@@ -360,6 +360,11 @@ class RoutePageControlPanel extends React.Component {
     const { patternId } = match.params;
     const { config } = this.context;
 
+    const routePdfURL = config.URL.ROUTE_PDF?.replace(
```

Comment:
Note: The notation used in `ROUTE_PDF` looks like it's a [RFC 6570 URI Template](https://datatracker.ietf.org/doc/html/rfc6570), but it isn't, because this `.replace()` call doesn't handle all variants (e.g. `{?foo,bar}`). We could consider a proper implementation like [`url-template`](https://github.com/bramstein/url-template).